CMakeLists.txt

```diff
@@ -4,7 +4,51 @@ project (prplmesh
          LANGUAGES C CXX
          VERSION 1.4.0)
 
-include(cmake/multiap-helpers.cmake)
+if(NOT WIN32)
```

Comment:
Why not simply include once from the top cmake?
Not sure there's a reason to include it multiple times after we switched to a single cmake project.

Reply:
I'll remove the first commit. The main purpose was to avoid including it multiple times (with different PROJECT values), but it's true that this was not the best approach.